src/main.ts

```diff
@@ -1,6 +1,6 @@
 import 'bootstrap';
-import {Settings} from 'settings';
-import {ThemeManager} from 'shared/theme-manager';
+import {Settings} from './settings';
```

Comment:
Some configuration is required when you use absolute paths, so lets use relative paths for now